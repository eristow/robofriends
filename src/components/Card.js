import React from 'react';

const Card = ({ id, name, email }) => {
  const imgStyle = {
    height: '200px',
    width: '200px',
  };

  return (
    <div className="tc bg-light-green dib br3 pa3 ma2 grow bw2 shadow-5">
      <img
        src={`https://robohash.org/${id}?size=200x200`}
        alt="robot"
        style={imgStyle}
      />
      <div>
        <h2>{name}</h2>
        <p>{email}</p>
      </div>
    </div>
  );
};

export default Card;
