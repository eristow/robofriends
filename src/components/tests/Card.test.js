import React from 'react';
import { shallow } from 'enzyme';
import Card from '../Card';

describe('Card', () => {
  it('should render Card component', () => {
    expect(shallow(<Card />)).toMatchSnapshot();
  });
});
