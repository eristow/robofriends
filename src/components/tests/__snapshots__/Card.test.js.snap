// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card should render Card component 1`] = `
<div
  className="tc bg-light-green dib br3 pa3 ma2 grow bw2 shadow-5"
>
  <img
    alt="robot"
    src="https://robohash.org/undefined?size=200x200"
    style={
      Object {
        "height": "200px",
        "width": "200px",
      }
    }
  />
  <div>
    <h2 />
    <p />
  </div>
</div>
`;
